src/ripple/core/impl/Config.cpp

```diff
@@ -162,7 +162,12 @@ void Config::setup (std::string const& strConf, bool bQuiet)
     // config directory and that with "db" as the data
     // directory.
 
-    QUIET = bQuiet;
+    QUIET = bQuiet || bSilent;
+    SILENT = bSilent;
+    RUN_STANDALONE = bStandalone;
+
+    if (RUN_STANDALONE)
+        LEDGER_HISTORY = 0;
```

Comment:
Good catch. The original behavior had this after the `load`, and I missed it in cut/paste. Will fix.